pkg/backend/display/progress.go

```diff
@@ -1196,30 +1197,6 @@ func (display *ProgressDisplay) getStepInProgressDescription(step engine.StepEve
 	return op.Color() + getDescription() + colors.Reset
 }
 
-func writePropertyKeys(b *bytes.Buffer, propMap resource.PropertyMap, op deploy.StepOp) {
-	if len(propMap) > 0 {
-		writeString(b, " ")
-		writeString(b, op.Prefix())
-
-		keys := make([]string, 0, len(propMap))
-		for k := range propMap {
-			keys = append(keys, string(k))
-		}
-		sort.Strings(keys)
-
-		index := 0
-		for _, k := range keys {
-			if index != 0 {
-				writeString(b, ",")
-			}
-			writeString(b, k)
-			index++
-		}
-
-		writeString(b, colors.Reset)
-	}
-}
```

Comment:
what happened to this.  does this mean when a resource changes, we don't print the keynames for what chnaged?  i personally found that useful.